examples/fully-private-eks-cluster/README.md

```diff
@@ -125,7 +125,7 @@ Create a Terraform variable definition file called base.tfvars and update the va
 
 ```shell script
 region             = "us-west-2"
-cluster_version    = "1.22"
+cluster_version    = "1.23"
```

Comment:
should be 1.24

Reply:
@FernandoMiguel I don't think 1.24 is supported yet.
https://github.com/aws-ia/terraform-aws-eks-blueprints/issues/1177

I would like to change the input variable default to 1.24 if it is supported.
https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/examples/fully-private-eks-cluster/eks/variables.tf#L4

Reply:
i've been running 1.24 with a few blueprints addons without issues